src/game.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+class Game {
+  constructor() {
+    this.frame = null;
+    this.turn = 0;
+    this.scorecard = [];
+    this.MAXIMUM_FRAME = 10;
+  }
+
+  beginFrame() {
+    if(this.turn >= this.MAXIMUM_FRAME) {
+      return this.finishGame();
+    } 
+    this.turn += 1;
+    this.frame = new Frame(this.turn);
+    return `Frame ${this.turn}`;
+  }
+
+  throwBall(points) {
+    return this.frame.addRoll(points);
+  }
+
+  addToScorecard() {
+    this.scorecard.push(this.frame.score);
+    return this.scorecard;
+  }
+
+  strikeBonus() {
```

Comment:
This function and the one below could do with some comments explaining the logic as it isn't obvious to someone unfamiliar with bowling.

Reply:
Ok noted, thank you for the advice.